apps/dashboard/src/components/workflow-editor/steps/shared/configure-preview-accordion.tsx

```diff
@@ -0,0 +1,79 @@
+import { CSSProperties, useEffect, useRef, useState } from 'react';
+import { Code2 } from '@/components/icons/code-2';
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@/components/primitives/accordion';
+import { Button } from '@/components/primitives/button';
+import { Editor } from '@/components/primitives/editor';
+import { loadLanguage } from '@uiw/codemirror-extensions-langs';
+
+const extensions = [loadLanguage('json')?.extension ?? []];
+
+interface ConfigurePreviewAccordionProps {
+  editorValue: string;
+  setEditorValue: (value: string) => void;
+  previewStep: () => void;
+}
+
+export const ConfigurePreviewAccordion = ({
+  editorValue,
+  setEditorValue,
+  previewStep,
+}: ConfigurePreviewAccordionProps) => {
+  const [accordionValue, setAccordionValue] = useState<string | undefined>('payload');
```

Comment:
What do we initiate this to `payload`? Shouldn't it be a `'{}'`?

Reply:
Nope, it's opened accordion (there is only one called payload), when set to undefined, the accordion closes

Reply:
The one you refer to is the default editor value